lib/matplotlib/offsetbox.py

```diff
@@ -1205,6 +1225,7 @@ def __init__(self, arr,
                  **kwargs
                  ):
 
+        OffsetBox.__init__(self)
```

Comment:
Because the `self._stale` attribute is added in the parent classes` __init__` and some of the `set_*` methods used in this `__init__` require it to be there so the parent class must be inited first.